README.md

```diff
@@ -1,6 +1,9 @@
 ### How to use this repository?
-https://github.com/Homebrew/brew/blob/master/docs/brew-tap.md
-
+```
+brew tap robertgzr/tap
```

Comment:
Maybe just replacing the dead link with https://github.com/Homebrew/brew/blob/master/docs/Taps.md would be better?

My reasoning is that anybody that randomly stumbles into this repo can also learn what it is there...

Reply:
I think we must leave `` `brew tap robertgzr / tap``. And example. It's really hard for users.